__mocks__/handlers/standup.handler.ts

```diff
@@ -0,0 +1,33 @@
+import { rest } from 'msw';
+const URL = process.env.NEXT_PUBLIC_BASE_URL;
+
+const standupHandler = [
+    rest.post(`${URL}/progresses`, async (req, res, ctx) => {
+        const body = await req.json();
+        return res(
+            ctx.status(200),
+            ctx.json({
+                message: 'User Progress document created successfully.',
+                data: body,
+            })
+        );
+    }),
+    rest.get(`${URL}/progresses`, async (req, res, ctx) => {
+        return res(ctx.status(201), ctx.json({ data: [] }));
```

Comment:
ctx.status(201) status code is used to indicate that the request was successful and that a new resource was created. However, the get handler is not creating a new resource, so it should return the ctx.status(200) status code instead

Reply:
fixed

Reply:
fixed